coredns/assets/configuration/spec.yaml

```diff
@@ -31,9 +31,9 @@ files:
     options: []
   - template: instances
     options:
-    - name: openmetrics_endpoint
+    - name: prometheus_url
```

Comment:
Even though `prometheus_url` is the default and enabled, should we also still add `openmetrics_url` just as an optional parameter in case a customer using `auto_conf.yaml` wanted to use V2?

Reply:
I'm not sure if customers typically modify the auto_conf.yaml, but I can add it as an optional parameter. I'll check with the containers team.

Reply:
I've added it as an option parameter